kanin/src/app.rs

```diff
@@ -112,7 +120,28 @@ impl App {
     /// # Panics
     /// On connection errors after the initial connection is established, the app will simply panic.
     pub async fn run(self, amqp_addr: &str) -> Result<()> {
```

Comment:
This function's doc comment now looks very similar to `run_with_connection`. I think we should just boil it down to something like "Connects to AMQP with the given address, then calls `run_with_connection`", and the user can just look at `run_with_connection` for the docs.

Reply:
That was on purpose. Since you'll only ever use one of them, I think it makes sense that you can just look at the docs for the one you are using and don't have to jump around to find them. I don't feel too strongly about it though.

Reply:
I think we shouldn't repeat ourselves. If we just have the docs in `run_with_connection` then we don't have to update multiple places if we change something.